modules/log6x-logging-loki-cli-install.adoc

```diff
@@ -0,0 +1,209 @@
+// Module is included in the following assemblies:
+//
+// * observability/logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-logging-loki-cli-install_{context}"]
+= Installing the {loki-op} by using the CLI
+
+To install and configure logging on your {product-title} cluster, you must first install {loki-op}, which manages the log store `Loki`. `Loki` log store is deployed and configured by the {loki-op} by configuring the resource `LokiStack`. This can be done from the {product-title} command-line interface (CLI).
```

Comment:
```suggestion
To install and configure logging on your {product-title} cluster, you must first install {loki-op}, which manages the log store `Loki`. `Loki` log store is deployed and configured by the {loki-op} by reconciling the resource `LokiStack`. This can be done from the {product-title} command-line interface (CLI).
```